packages/developer-console-spa/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "developer-console-spa",
+  "version": "0.1.0",
+  "homepage": "/console",
+  "private": true,
```

Comment:
Include Author and maintainer information

Reply:
Updated.